jsapp/js/components/table.es6

```diff
@@ -133,22 +133,31 @@ export class DataTable extends React.Component {
     if (originalRow._validation_status.uid) {
       return VALIDATION_STATUSES[originalRow._validation_status.uid];
     } else {
-      return null;
+      return VALIDATION_STATUSES.no_status;
     }
   }
   onValidationStatusChange(sid, index, evt) {
     const _this = this;
-    const data = {'validation_status.uid': evt.value};
-    dataInterface.updateSubmissionValidationStatus(_this.props.asset.uid, sid, data).done((result) => {
-      if (result.uid) {
-        _this.state.tableData[index]._validation_status = result.uid;
+
+    if (evt.value === null) {
+      dataInterface.removeSubmissionValidationStatus(_this.props.asset.uid, sid).done((result) => {
+        _this.state.tableData[index]._validation_status = {};
         _this.setState({tableData: _this.state.tableData});
-      } else {
-        console.error('error updating validation status');
-      }
-    }).fail((error)=>{
-      console.error(error);
-    });
+      }).fail((error)=>{
```

Comment:
since `console.error` is functionally identical to `(error)=>{ console.error(error); }`, you can just do 
`.fail(console.error)`

You can also do that below

Reply:
nice! fixed